arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -488,6 +511,18 @@
         }
       }
     },
+    "CommitmentPlanPatchPayload": {
+      "type": "object",
+      "description": "The properties of a commitment plan which may be updated via PATCH.",
+      "properties": {
+        "tags": {
+          "$ref": "#/definitions/Tags"
```

Comment:
Please provide description for both the properties. The description over here is the documentation of the referenced object is used in this context.